OgreMain/src/OgreRenderSystem.cpp

```diff
@@ -240,28 +240,20 @@ namespace Ogre {
     //-----------------------------------------------------------------------
     void RenderSystem::_initRenderTargets(void)
     {
-
-        // Init stats
-        for(
-            RenderTargetMap::iterator it = mRenderTargets.begin();
-            it != mRenderTargets.end();
-            ++it )
+        for(auto &&a : mRenderTargets)
```

Comment:
please use `auto& p` when iterating maps.
- The name (pair) should correspond to the element type.
- rvalue at this place is confusing and has no advantage